cdap-data-fabric/src/main/java/org/apache/tephra/TxConstants.java

```diff
@@ -161,11 +162,11 @@
     /**
      * The maximum time in seconds that a transaction can be used for data writes.
      */
-    public static final String CFG_TX_MAX_LIFETIME = "data.tx.max.lifetime";
+    public static final String CFG_TX_MAX_LIFETIME = Constants.Tephra.CFG_TX_MAX_LIFETIME;
```

Comment:
Why need to copy this and the next constants?? We can use the one from Tephra directly, right? (Also I don't see usage in this PR).